spec/helpers/text_formatting_helper_spec.rb

```diff
@@ -0,0 +1,17 @@
+require "rails_helper"
+
+describe TextFormattingHelper, type: :helper do
+  describe "#safe_format" do
+    subject { safe_format content }
+
+    context "with new lines" do
+      let(:content) { "hello\r\n\r\nworld" }
+      it { is_expected.to eql "<p>hello</p>\n\n<p>world</p>" }
+    end
+
+    context "with html content" do
+      let(:content) { "<strong>hello</strong> <em>world</em>" }
```

Comment:
Do we need to configure `ActionView::Base.sanitized_allowed_tags` to allow the tags used in the privacy policy (I actually thought it allowed `strong` and `em` by default)? It looks like `ul` and `li` will be needed, but if we're including some of the content in different sections as well we may need to allow headings too?

Reply:
Hah - yes, good call - the actual helper I'd taken from School Experience - forgot I'd added that bit.

Reply:
At present there's not richer formatting anyway - its all just plain text, if its decided to use HTML we could switch out `strip_tags` with `sanitize` but when I talked to Myles earlier it just sounds like it needs the white space to be shown in the output